diff -up photoqt-3.1/cplusplus/scripts/handlingfacetags.cpp.omv~ photoqt-3.1/cplusplus/scripts/handlingfacetags.cpp
--- photoqt-3.1/cplusplus/scripts/handlingfacetags.cpp.omv~	2023-05-18 05:27:34.239693808 +0200
+++ photoqt-3.1/cplusplus/scripts/handlingfacetags.cpp	2023-05-18 05:27:55.451921501 +0200
@@ -57,7 +57,7 @@ QVariantList PQHandlingFaceTags::getFace
         image->readMetadata();
     } catch (Exiv2::Error& e) {
         // An error code of 11 means image not supported. This is much more reliable than, e.g., checking a file ending
-        if(e.code() != 11)
+        if(e.code() != Exiv2::ErrorCode::kerFileContainsUnknownImageType)
             LOG << CURDATE << "PQHandlingFaceTags::getFaceTags() - ERROR reading metadata (caught exception): " << e << NL;
         else
             DBG << CURDATE << "PQHandlingFaceTags::getFaceTags() - ERROR reading metadata (caught exception): " << e << NL;
diff -up photoqt-3.1/cplusplus/scripts/metadata.cpp.omv~ photoqt-3.1/cplusplus/scripts/metadata.cpp
--- photoqt-3.1/cplusplus/scripts/metadata.cpp.omv~	2023-05-18 05:26:20.849898921 +0200
+++ photoqt-3.1/cplusplus/scripts/metadata.cpp	2023-05-18 05:27:21.738559205 +0200
@@ -120,7 +120,7 @@ void PQMetaData::updateMetadata(QString
     } catch (Exiv2::Error& e) {
         // An error code of 11 means unknown file type
         // Since we always try to read any file's meta data, this happens a lot
-        if(e.code() != 11)
+        if(e.code() != Exiv2::ErrorCode::kerFileContainsUnknownImageType)
             LOG << CURDATE << "PQMetaData::updateMetadaya(): ERROR reading exiv data (caught exception): " << e.what() << NL;
         else
             DBG << CURDATE << "PQMetaData::updateMetadaya(): ERROR reading exiv data (caught exception): " << e.what() << NL;
